src/genophenocorr/preprocessing/_vep.py

```diff
@@ -80,6 +80,18 @@ def annotate(self, variant_coordinates: VariantCoordinates) -> typing.Sequence[T
 
         return annotations
 
+    def _parse_variant_effect(self, effect: str) -> typing.Optional[VariantEffect]:
+        if effect.upper() == "5_PRIME_UTR_VARIANT":
```

Comment:
I have two minor suggestions:
-  please refactor such that you do the conversion to uppercase at most once per method body instead of 3 times in the worst case.
- logging, as currently written, will construct the string unconditionally, regardless of the set log level threshold (e.g. even if the user sets the threshold to see ERRORs only). This is because you're submitting the f-string which should be avoided when logging a statement. Instead, we should do something like this:
  ```python
  self._logging.warning('Variant effect %s was not found ...', effect)
  ```
  Using this form, the logging framework will only create the final string if the logging threshold is low enough (WARN or lower in this case).